test/index.spec.ts

```diff
@@ -101,3 +106,37 @@ describe('CreateUser Mutation Test', () => {
     expect(hasInvalidPasswordError).to.be.true;
   });
 });
+
+//Temporary test for login, it compares to a specific return, it will be changed in the next PR's.
+describe('Login Mutation test - Return', () => {
+  it('Should return be equal input', async () => {
+    const query = `mutation($email: String!, $password: String!) {
+      login(email: $email, password: $password) {
+        user {
+          id
+          name
+          email
+          birthdate
+        },
+        token
+      }
+    }`;
+
+    const email = 'mochauser@email.com';
+    const password = 'mochauserPassword1';
+
+    const result = await connection.post('/graphql', {
+      query,
+      variables: { email, password },
+    });
+
+    const loginResponse = {
+      id: 12,
+      name: 'Rodrigo',
+      email: 'rodrigo@email.com',
+      birthdate: '01-01-1980',
+    };
+
+    expect(JSON.stringify(result.data.data.login.user)).to.be.equal(JSON.stringify(loginResponse));
```

Comment:
você pode usar um `expect(result.data.data.login.user).to.be.deep.eq({ ... }) para não precisar do JSON.stringify

Reply:
Sim, eu mexi nesse teste no próximo PR, assim que atualizar o próximo vai ser corrigido.

Reply:
Só uma sutil diferença: em JSONs em geral, a ordem dos campos não importam, porém ainda assim o Javascript considera ordem na montagem de objetos. Uma comparação com `JSON.stringify` acabaria sendo considerada errada caso os objetos tivessem os mesmos campos e mesmo valores, mas em ordens diferentes. E uma comparação com `deep.eq` seria considerada correta, mesmo em ordem diferente.

O segundo cenário é o mais satisfatório pra gente, dado que não queremos distinguir ordem de chaves nos objetos.